cmd/edit-script.go

```diff
@@ -0,0 +1,76 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/twpayne/chezmoi/lib/chezmoi"
+	vfs "github.com/twpayne/go-vfs"
+)
+
+var editScriptsCmd = &cobra.Command{
+	Use:   "edit-scripts scripts",
+	Args:  cobra.MinimumNArgs(1),
+	Short: "Edit scripts in the scripts folder",
+	RunE:  makeRunE(config.runEditScriptCmd),
+}
+
+type editScriptsCmdConfig struct {
+	run    bool
+	prompt bool
+}
+
+func init() {
+	rootCmd.AddCommand(editScriptsCmd)
+
+	persistentFlags := editScriptsCmd.PersistentFlags()
+	persistentFlags.BoolVarP(&config.editScripts.run, "run", "r", false, "run after editing")
+	persistentFlags.BoolVarP(&config.editScripts.prompt, "prompt", "p", false, "prompt before running")
+}
+
+func (c *Config) runEditScriptCmd(fs vfs.FS, args []string) error {
+	ts, err := c.getTargetState(fs)
+	if err != nil {
+		return err
+	}
+	//entries, err := c.getEntries(ts, args)
+	var scripts []*chezmoi.Script
+	for _, arg := range args {
+		if s, ok := ts.Scripts[chezmoi.StripTemplateExtension(arg)]; ok {
+			scripts = append(scripts, s)
+		}
+	}
+	println(scripts)
```

Comment:
This looks like debug code. Please remove it.

Reply:
That's for running all remaining scripts at the end. When you select 'a', that's what get's invoked.
I've cleaned it up a little, I hope its more readable now.